semgrep_output_v1.atd

```diff
@@ -2167,6 +2169,37 @@ type resolution_result = [
   | ResolutionError of resolution_error list
 ]
 
+(* ----------------------------- *)
+(* SCA part 4: Symbol analysis *)
+(* ----------------------------- *)
+
+(* "Symbol analysis" is about determining the third-party functions which
+   are used by a given project.
+
+   We may be interested in this for the purposes of analyzing the vulnerability
+   of the repo based on its dependencies, and providing more customized/informed
+   guidance on how to remediate SCA vulns.
+
+   For instance, source code like
+   ```
+   from werkzeug.wrappers import Request
+   ```
+   may be described as using the "symbol" [werkzeug, wrappers, Request].
```

Comment:
There is some PRIVACY issue here; we're not sending the code of the customer but the symbols/libs it used
so be cautious here.